packages/utils/src/signatures.ts

```diff
@@ -21,7 +21,7 @@ export async function verifySignature(
   // Determine if this signature is from an EOA or a contract.
   switch (cacaoSignature.t) {
     case "eip191":
-      return isValidEip191Signature(address, reconstructedMessage, cacaoSignature.s);
+      return await isValidEip191Signature(address, reconstructedMessage, cacaoSignature.s);
```

Comment:
would this break current integrations ? (non promise => promise)

Reply:
nope, [the main util](https://github.com/WalletConnect/walletconnect-monorepo/pull/5655/files/46e50d83f407ecda40cac4a0902734615204c22b#diff-db0cda938ae799e2adeabde139490497ab87cade5efd8e1072f367a77eb2b274R13) is still async